src/browser_action/popup.html

```diff
@@ -62,35 +57,6 @@ <h1>Metrics</h1>
     </main>
 
     <footer class="metadata">
-
-      <div id="legend">
```

Comment:
I'm not sure about removing this.

It means no explanation of what the blue dots are, or the beaker. I also think we should deprecate FID (maybe just reusing the grey circle icon as can't think of a better one).

<img width="559" alt="image" src="https://github.com/GoogleChrome/web-vitals-extension/assets/10931297/f4554840-62a6-448e-a35a-c0fa909b2709">

Reply:
WDYT about only showing the 3 CWV by default, and adding opt-in settings for visualizing other Web Vitals and deprecated metrics? We can keep logging them to the console, or gate them behind the settings too. I think that solves the iconography/legend problem because developers would consciously add new classes of metrics as needed, it makes the HUD and popup more consistent (the HUD doesn't have any icons right now), and it reverts back to the cleaner UI before #131.

Reply:
Personally I'm not sure it's worth the effort. But then I don't use the pop over much (and ever more rarely the HUD!).

To me the pop over and HUD covers so much of the screen anyway you quickly want to dismiss it anyway, so why not show everything rather than the complexity of a two-state popover and HUD? 

I'm not aware of any complaints since adding the three extra metrics (quite the opposite in fact as [the Tweet announcing it got a bit of traction](https://twitter.com/tunetheweb/status/1679472895410544642)) and other tools like PSI show them by default too.

Plus we recommend FCP and TTFB as diagnostic metrics to understand LCP so I don't see why we'd want to hide them, now they are here.

Also shouldn't we somehow deprecate FID anyway to indicate it will be removed from here at some point?

I agree it's not the most beautiful design but I'm always one for functionality over form (mostly cause I'm a poor designer! 😁). If you did make them optional then would you indicate the 3 CWVs differently when all 6 were shown? Maybe we should just remove the explanation icons completely, and then the key, (like the HUD) rather than removing the extra metrics? But either way it sounds like a separate PR to this one.

Reply:
> Maybe we should just remove the explanation icons completely, and then the key, (like the HUD) rather than removing the extra metrics? But either way it sounds like a separate PR to this one.

SGTM. PTAL https://github.com/GoogleChrome/web-vitals-extension/pull/171